tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -391,16 +393,18 @@ user edits an external methodology
 
     ${accordion}=    user opens publication on the admin dashboard    ${publication}
     user clicks link    Edit external methodology    ${accordion}
-    user waits until page contains title    Edit external methodology link
+    user waits until page contains title    Edit external methodology link    10
```

Comment:
I think, since `${timeout}` is set to 30 by default, that this will actually decrease the wait time on this keyword, which I assume isn't the intention here?